### ISSUE TITLE
can not import neutcurve

### ISSUE THREAD

**Issue by zxqedu** [jbloomlab/neutcurve#21]
>>> import neutcurve
/Users/niulab/miniconda/envs/py3/lib/python3.8/site-packages/dmslogo/logo.py:38: MatplotlibDeprecationWarning: 
The createFontList function was deprecated in Matplotlib 3.2 and will be removed two minor releases later. Use FontManager.addfont instead.
  matplotlib.font_manager.createFontList(
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/niulab/miniconda/envs/py3/lib/python3.8/site-packages/neutcurve/__init__.py", line 22, in <module>
    from neutcurve.curvefits import CurveFits  # noqa: F401
  File "/Users/niulab/miniconda/envs/py3/lib/python3.8/site-packages/neutcurve/curvefits.py", line 12, in <module>
    import dmslogo.facet
  File "/Users/niulab/miniconda/envs/py3/lib/python3.8/site-packages/dmslogo/__init__.py", line 23, in <module>
    from dmslogo.logo import draw_logo  # noqa: F401
  File "/Users/niulab/miniconda/envs/py3/lib/python3.8/site-packages/dmslogo/logo.py", line 38, in <module>
    matplotlib.font_manager.createFontList(
  File "/Users/niulab/miniconda/envs/py3/lib/python3.8/site-packages/matplotlib/cbook/deprecation.py", line 234, in wrapper
    return func(*args, **kwargs)
  File "/Users/niulab/miniconda/envs/py3/lib/python3.8/site-packages/matplotlib/font_manager.py", line 606, in createFontList
    prop = ttfFontProperty(font)
  File "/Users/niulab/miniconda/envs/py3/lib/python3.8/site-packages/matplotlib/font_manager.py", line 464, in ttfFontProperty
    weight = int(get_weight())
  File "/Users/niulab/miniconda/envs/py3/lib/python3.8/site-packages/matplotlib/font_manager.py", line 447, in get_weight
    font.get_ps_font_info()["weight"].replace(" ", "") or "")

**Comment by jbloom** [comment]
Hmmm, I get the deprecation warning (at some point I need to update that code), but not an actual error---and it still runs fine for me. Does the import actually fail with an error for you?

If so, can you provide details on how exactly you install `matplotlib` and `dmslogo` to help me troubleshoot?

**Comment by zxqedu** [comment]
Actually failed to import, below for your reference.
________________
(py3) bogon:数据处理 niulab$ pip install dmslogo
Requirement already satisfied: dmslogo in /Users/niulab/miniconda/envs/py3/lib/python3.8/site-packages (0.3.2)
Requirement already satisfied: numpy>=1.13 in /Users/niulab/miniconda/envs/py3/lib/python3.8/site-packages (from dmslogo) (1.19.1)
Requirement already satisfied: pandas>=0.23 in /Users/niulab/miniconda/envs/py3/lib/python3.8/site-packages (from dmslogo) (1.1.1)
Requirement already satisfied: setuptools in /Users/niulab/miniconda/envs/py3/lib/python3.8/site-packages (from dmslogo) (49.6.0.post20200814)
Requirement already satisfied: matplotlib>=3.0.0 in /Users/niulab/miniconda/envs/py3/lib/python3.8/site-packages (from dmslogo) (3.3.1)
Requirement already satisfied: pytz>=2017.2 in /Users/niulab/miniconda/envs/py3/lib/python3.8/site-packages (from pandas>=0.23->dmslogo) (2020.1)
Requirement already satisfied: python-dateutil>=2.7.3 in /Users/niulab/miniconda/envs/py3/lib/python3.8/site-packages (from pandas>=0.23->dmslogo) (2.8.1)
Requirement already satisfied: cycler>=0.10 in /Users/niulab/miniconda/envs/py3/lib/python3.8/site-packages (from matplotlib>=3.0.0->dmslogo) (0.10.0)
Requirement already satisfied: certifi>=2020.06.20 in /Users/niulab/miniconda/envs/py3/lib/python3.8/site-packages (from matplotlib>=3.0.0->dmslogo) (2020.6.20)
Requirement already satisfied: pillow>=6.2.0 in /Users/niulab/miniconda/envs/py3/lib/python3.8/site-packages (from matplotlib>=3.0.0->dmslogo) (7.2.0)
Requirement already satisfied: kiwisolver>=1.0.1 in /Users/niulab/miniconda/envs/py3/lib/python3.8/site-packages (from matplotlib>=3.0.0->dmslogo) (1.2.0)
Requirement already satisfied: pyparsing!=2.0.4,!=2.1.2,!=2.1.6,>=2.0.3 in /Users/niulab/miniconda/envs/py3/lib/python3.8/site-packages (from matplotlib>=3.0.0->dmslogo) (2.4.7)
Requirement already satisfied: six>=1.5 in /Users/niulab/miniconda/envs/py3/lib/python3.8/site-packages (from python-dateutil>=2.7.3->pandas>=0.23->dmslogo) (1.15.0)

**Comment by jbloom** [comment]
This is strange. I can't reproduce error, which makes me think it is some installation difference that makes it hard to troubleshoot. Can you try installing into a clean conda environment and see if you still get the error:
```
conda env create --name dmslogo_test python=3.8 pip
conda create --name dmslogo_test python=3.8 pip
conda activate dmslogo_test
pip install dmslogo
```

Then try opening Python in that new conda environment and seeing if you can import `dmslogo`. This works on my computer, so if it still fails on yours we can try to troubleshoot more from there.

**Comment by zxqedu** [comment]
Thanks for your help! I will try it and send feedback. -------- 原始邮件 --------发件人： Jesse Bloom <notifications@github.com>日期： 2020年8月29日周六 22:34收件人： jbloomlab/neutcurve <neutcurve@noreply.github.com>抄送： zxqedu <zhuxiangqian@nibs.ac.cn>, Author <author@noreply.github.com>主    题： Re: [jbloomlab/neutcurve] can not import neutcurve (#21)
This is strange. I can't reproduce error, which makes me think it is some installation difference that makes it hard to troubleshoot. Can you try installing into a clean conda environment and see if you still get the error:
conda env create --name dmslogo_test python=3.8 pip
conda create --name dmslogo_test python=3.8 pip
conda activate dmslogo_test
pip install dmslogo

Then try opening Python in that new conda environment and seeing if you can import dmslogo. This works on my computer, so if it still fails on yours we can try to troubleshoot more from there.

**Comment by zxqedu** [comment]
In Conda environment, it works well.
Great!

**Comment by jbloom** [comment]
Great.

**Comment by jbloom** [comment]
Should be fixed in #30, which is version 0.5.5.
